mkdocs/utils.py

```diff
@@ -35,6 +36,39 @@
     text_type = unicode         # noqa
 
 
+def yaml_load(source, loader=yaml.Loader):
+    """
+    Custom yaml loader.
+
+    Load all strings as unicode.
+    http://stackoverflow.com/a/2967461/3609487
+    """
+
+    class Loader(loader):
+
+        """Custom Loader."""
+
+        pass
```

Comment:
I explain a little further.  If I remove the pass, Python will think everything under the class is supposed to be part of it and complain about unexpected indentation (or something similar).  Pass tells python "it's okay, there's nothing else here".